unittests/test_utility.py

```diff
@@ -1293,3 +1295,26 @@ def test_cray_cle_info_missing_parts(tmp_path):
     assert cle_info.date is None
     assert cle_info.network is None
     assert cle_info.patchset == '09'
+
+
+def test_jsonext_dump(tmp_path):
+    json_dump = tmp_path / 'test.json'
+    json_dump_indent = tmp_path / 'test_indent.json'
+    with open(json_dump, 'w') as f:
+        jsonext.dump({'foo': sn.defer(['bar'])}, f)
+
+    with open(json_dump, 'r') as f:
+        assert '{"foo": ["bar"]}' == f.read()
+
+    with open(json_dump_indent, 'w') as f:
+        jsonext.dump({'foo': sn.defer(['bar'])}, f, indent=2)
```

Comment:
Better pass `separators=(',', ':')` instead of `indent=2` so that your assertion below becomes much more compact.